src/ripple/app/ledger/impl/LedgerConsensus.cpp

```diff
@@ -1522,20 +1522,20 @@ class LedgerConsensusImp
                 << mPeerPositions.size () << " nw:" << neededWeight
                 << " thrV:" << threshVote << " thrC:" << threshConsensus;
 
-            for (auto it = closeTimes.begin ()
-                , end = closeTimes.end (); it != end; ++it)
+            for (auto i = closeTimes.begin ()
+                , end = closeTimes.end (); i != end; ++it)
```

Comment:
It depends on the implementation.  There could be a minor one if the implementation does a log(N) search for `end()`.  I think the standard forbids this, and libc++ doesn't.  Not positive about gcc and VS.

Glancing through this loop, this loop looks like a good opportunity to for a for-range:
````
for (auto const& i : closeTimes)
````
Doing so would also cache `end()`.